integration_tests/e2e/prisons/configuration/prisonConfig.cy.ts

```diff
@@ -0,0 +1,91 @@
+import TestData from '../../../../server/routes/testutils/testData'
+import HomePage from '../../../pages/home'
+import Page from '../../../pages/page'
+import SupportedPrisonsPage from '../../../pages/prisons/SupportedPrisons'
+import ViewSessionTemplatesPage from '../../../pages/prisons/sessionTemplates/viewSessionTemplates'
+import PrisonConfigPage from '../../../pages/prisons/configuration/prisonConfig'
+import { SessionTemplatesRangeType } from '../../../../server/data/visitSchedulerApiTypes'
+
+context('Prison configuration', () => {
+  const prisonCode = 'HEI'
+  const rangeType: SessionTemplatesRangeType = 'CURRENT_OR_FUTURE'
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+    cy.task('stubPrisons')
+    cy.task('stubGetAllPrisons')
+    cy.signIn()
+
+    cy.task('stubGetPrisonContactDetails', { prisonCode })
+  })
+
+  describe('Prison contact details', () => {
```

Comment:
No integration tests yet for add/edit contact details. Will do as a separate PR.